packages/dd-trace/test/appsec/telemetry.spec.js

```diff
@@ -315,6 +432,44 @@ describe('Appsec Telemetry metrics', () => {
         expect(duration).to.be.eq(66)
         expect(durationExt).to.be.eq(77)
       })
+
+      it('should sum rasp.duration and rasp.durationExt with telemetry enabled and metrics disabled', () => {
+        appsecTelemetry.enable({
+          enabled: true,
+          metrics: false
+        })
+
+        appsecTelemetry.updateRaspRequestsMetricTags({
+          duration: 42,
+          durationExt: 52
+        }, req, 'rule_type')
+
+        appsecTelemetry.updateRaspRequestsMetricTags({
+          duration: 24,
+          durationExt: 25
+        }, req, 'rule_type')
+
+        const { raspDuration, raspDurationExt, raspEvalCount } = appsecTelemetry.getRequestMetrics(req)
+
+        expect(raspDuration).to.be.eq(66)
+        expect(raspDurationExt).to.be.eq(77)
+        expect(raspEvalCount).to.be.eq(2)
+      })
+
+      it('should not increment any metric if telemetry metrics are disabled', () => {
+        appsecTelemetry.enable({
+          enabled: true,
+          metrics: false
+        })
+
+        appsecTelemetry.updateWafRequestsMetricTags({
```

Comment:
```suggestion
        appsecTelemetry.updateRaspRequestsMetricTags({
```

Reply:
yes, I'll modify it.
And I realise that I wrote the tests of `updateRaspRequestsMetricTag` under `    describe('updateWafRequestMetricTags', () => {`, and I should've created new describe section, I'll modify it.